test/parallel/test-http2-endafterheaders.js

```diff
@@ -9,7 +9,7 @@ const Countdown = require('../common/countdown');
 
 const server = http2.createServer();
 server.on('stream', common.mustCall((stream, headers) => {
-  const check = headers[':method'] === 'GET' ? true : false;
+  const check = headers[':method'] === 'GET' ;
```

Comment:
```suggestion
  const check = headers[':method'] === 'GET';
```

Reply:
Does the linter not catch that? O_O

Reply:
I would hope so, but CI is currently unavailable.